include/ppx/grfx/vk/vk_command.h

```diff
@@ -157,6 +169,12 @@ class CommandBuffer
         uint32_t                          setCount,
         const grfx::DescriptorSet* const* ppSets);
 
+    void SetPushConstants(
+        const grfx::PipelineInterface* pInterface,
+        uint32_t                       count,
+        const void*                    pValues,
+        uint32_t                       offset);
```

Comment:
nit: dstOffset too?

Reply:
Thanks! Missed those and the related implementation. It's in now.